src/qt/dc.cpp

```diff
@@ -124,7 +126,10 @@ int wxQtDCImpl::GetDepth() const
 
 wxSize wxQtDCImpl::GetPPI() const
 {
-    return wxSize(m_qtPainter->device()->logicalDpiX(), m_qtPainter->device()->logicalDpiY());
+    QScreen *srn = QApplication::screens().at(0);
```

Comment:
> But screen count can be also zero.

see [QApplication crash if screen count is zero](https://bugreports.qt.io/browse/QTBUG-37876)